include/GafferSceneUI/SceneGadget.h

```diff
@@ -191,6 +193,16 @@ class GAFFERSCENEUI_API SceneGadget : public GafferUI::Gadget
 		/// Implemented to return the name of the object under the mouse.
 		std::string getToolTip( const IECore::LineSegment3f &line ) const override;
 
+		/// Saves a snapshot of `m_outputBuffer` to the given file. All formats supported by
```

Comment:
`m_outputBuffer` is a private member and a private class, so isn't really appropriate for API docs. Can we talk more generically about the rendered image, and explain that it's not available for the OpenGL renderer?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5533/commits/b6c2f5367845b1fe6f4d802a0ea795e2c95c9fa6